buildSrc/src/main/groovy/io.deephaven.java-jar-conventions.gradle

```diff
@@ -0,0 +1,17 @@
+plugins {
+    id 'java'
+}
+
+jar {
+    manifest {
+        attributes 'Implementation-Title': 'Deephaven Database',
+                'Implementation-Version': project.version,
+                'Provider': 'gradle'
+    }
+}
+
+tasks.withType(Jar).configureEach {
+    Jar jar ->
+        jar.preserveFileTimestamps = false
```

Comment:
Maybe a comment about how this makes the timestamps all ~epoch, so that jar files are binary-stable regardless of when they were built / edited / etc.